### ISSUE TITLE
Where to get?

### ISSUE THREAD

**Issue by jacobslusser** [perrybutler/rapid-server#1]
Is there source code and/or a binary for this project? Apologies if I'm missing something but I can't find any more than a README.md.

**Comment by Ahmad45123** [comment]
Same here.. 

Looks like its discontinued or something :/

**Comment by perrybutler** [comment]
Not a single soul had an interest in this until now. Since there was no rush, I never uploaded it. I certainly have some obligations to update my Github projects though. This one, and WP-OAuth could use some work. I appreciate your interest and I'll let you guys know when I post the source code.

**Comment by masroore** [comment]
Sounds interesting. Kindly push the source code.

**Comment by djordjeandrejevic** [comment]
+1, please upload it, I would love to do the benchmarking!

**Comment by PPaques** [comment]
Waiting here to test that on Windows CE devices

**Comment by masroore** [comment]
6 months had passed... no show...

**Comment by perrybutler** [comment]
It's uploaded enjoy :)

Make sure to edit the http.xml file to include a site/path/port. There is a default Site1 pointing to C:/site1 on port 30000.

P.S. Github for Windows sucks! Really bad!!! I had to use TortoiseGit just to get the commit to work, then I had to Sync thru Github for Windows because TortoiseGit is not straightforward with the Sync (where is the Sync or OK button? it just says Close). It's unacceptable for something that should be so simple. When I develop for WordPress, TortoiseSvn just WORKS.
